app/static/css/main.css

```diff
@@ -6,6 +6,10 @@ pre {
   white-space: pre-wrap;
 }
 
+codelike {
+  font-family: monospace, monospace;
```

Comment:
Listing monospace twice is redundant

Reply:
See here: https://stackoverflow.com/questions/38781089/font-family-monospace-monospace .

Reply:
Added.